### ISSUE TITLE
App not working with TalkBack

### ISSUE THREAD

**Issue by fareem1** [newsvoice/newsvoice#236]
Pointed out by a user with feedback - News category selection for TalkBack users is impossible since selection window fails to focus on categories and speaks several instances of 67%.

**Comment by vlidholt** [comment]
This should be fixed in a future update
